hazelcast/src/main/java/com/hazelcast/internal/metrics/impl/ProbeUtils.java

```diff
@@ -48,28 +51,25 @@ final class ProbeUtils {
     private static final Map<Class<?>, Integer> TYPES;
 
     static {
-        final Map<Class<?>, Integer> types = createHashMap(18);
+        final Map<Class<?>, Integer> types = createHashMap(20);
 
-        types.put(byte.class, TYPE_PRIMITIVE_LONG);
-        types.put(short.class, TYPE_PRIMITIVE_LONG);
-        types.put(int.class, TYPE_PRIMITIVE_LONG);
-        types.put(long.class, TYPE_PRIMITIVE_LONG);
+        Stream.of(byte.class, short.class, int.class, long.class).forEach(clazz -> types.put(clazz,
+                TYPE_PRIMITIVE_LONG));
 
-        types.put(Byte.class, TYPE_LONG_NUMBER);
-        types.put(Short.class, TYPE_LONG_NUMBER);
-        types.put(Integer.class, TYPE_LONG_NUMBER);
-        types.put(Long.class, TYPE_LONG_NUMBER);
-        types.put(AtomicInteger.class, TYPE_LONG_NUMBER);
-        types.put(AtomicLong.class, TYPE_LONG_NUMBER);
+        Stream.of(Byte.class, Integer.class, Short.class, Long.class, AtomicInteger.class, AtomicLong.class,
+                LongAdder.class, LongAccumulator.class).forEach(clazz -> types.put(clazz,
+                TYPE_LONG_NUMBER));
 
-        types.put(double.class, TYPE_DOUBLE_PRIMITIVE);
-        types.put(float.class, TYPE_DOUBLE_PRIMITIVE);
+        Stream.of(double.class, float.class).forEach(clazz -> types.put(clazz,
+                TYPE_DOUBLE_PRIMITIVE));
 
-        types.put(Double.class, TYPE_DOUBLE_NUMBER);
-        types.put(Float.class, TYPE_DOUBLE_NUMBER);
+        Stream.of(Double.class, Float.class).forEach(clazz -> types.put(clazz,
+                TYPE_DOUBLE_NUMBER));
```

Comment:
Completely nit-pick & optional: I personally find the previous approach to be more readable, and I don't see any tangible benefit from the `Stream` approach (if anything, slight perf loss, but obviously that's irrelevant in these circumstances).

Reply:
> Completely nit-pick & optional: I personally find the previous approach to be more readable, and I don't see any tangible benefit from the `Stream` approach (if anything, slight perf loss, but obviously that's irrelevant in these circumstances).

I think this one is a matter of opinion - repeating the same statement over and over again allows errors to creep in - this way you know that same operation (insert the `class` into `types` with the specified `TYPE`) is applied to every input `class` at a glance.

I would've happily left it alone but Checkstyle complained it'd got too long when I added the extra two :(

Reply:
Definitely a matter of opinion, just sharing my $0.02 🙂 

> I would've happily left it alone but Checkstyle complained it'd got too long when I added the extra two :(

Well in that case, the checkstyle gods have spoken 😅